crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -387,6 +383,87 @@ impl QueryBuilder {
         Ok(query)
     }
 
+    fn multi_get_events<'a>(
+        cursor: Option<(i64, i64)>,
+        descending_order: bool,
```

Comment:
As per our discussion yesterday, let's punt on the implementation of connection ordering until after MVP 2.0, which should slightly simplify this code.

Reply:
The PR fetches the StoredEvent instead of using IndexerReader's query_events_in_blocking_task which returns a SuiEvent. This allows us to change graphql's event representation. But yes in some stack of PRs it'd be better off being:

1. simple QueryBuilder query to fetch the raw StoredEvent, no filtering logic
2. add filtering logic

Separately but concurrently, fix the ordering